public/css/mixin/progress-bar.less

```diff
@@ -15,17 +15,22 @@
       padding: 1.25em 10%;
     }
 
+    .positioned {
+      position: absolute;
+    }
+
     .bubble {
       .rounded-corners(.25em);
       background-color: @body-bg-color;
       border: 1px solid @gray-light;
-      position: absolute;
-      bottom: 0;
+      position: relative;
       box-shadow: 0 0 1em 0 rgba(0, 0, 0, .1);
-      margin-left: -4em;
       padding: .25em .5em;
       text-align: center;
-      width: 8em;
+      width: auto;
+      // The wrapper of .bubble is dynamically moved to the left based on the value of the progress bar
+      // This moves the center of the bubble to the beginning of the wrapper regardless of the size of the content.
+      transform: translate(-50%, 0);
```

Comment:
This doesn't seem to work properly, yet, when the bubbles are `.right` aligned like it is e.g. applied in `DowntimeCard.php` lines 162 and 145.

This is done, when start and end bubbles are close together.

This is why the bubbles have fixed widths.

![Screen Shot 2021-09-28 at 13 14 56](https://user-images.githubusercontent.com/6977434/135076758-404ea19c-b261-4691-b471-4c2b682782e7.png)

Reply:
Should work with the latest changes 😄 .

Reply:
Leaving out the left and right classes isn't the solution, sorry. Please restore the classes for preventing the bubbles to overlap.

![Screen Shot 2021-09-28 at 17 32 28](https://user-images.githubusercontent.com/6977434/135118646-79630714-d4f2-435f-bf09-c2380ccc2260.png)

In this example you see that problem with the obscured bubble.